CMakeLists.txt

```diff
@@ -199,20 +207,28 @@ PKG_CONFIG_APPEND_LIBS(roboptim-core)
 # Search for dependencies.
 SET(BOOST_COMPONENTS date_time filesystem system unit_test_framework)
 SEARCH_FOR_BOOST()
-ADD_REQUIRED_DEPENDENCY("eigen3 >= 3.2.0")
-ADD_REQUIRED_DEPENDENCY("liblog4cxx >= 0.10.0")
+SEARCH_FOR_EIGEN("eigen3 >= 3.2.0")
```

Comment:
Unlike Add_required_dependency, search_for_eigen does not add the eigen compile flags (or the dependency with the eigen3.PC file) in roboptim-core.pc

Without that the compilation of the plugins won't work.